docs/sources/administration/user-management/manage-org-users/index.md

```diff
@@ -38,6 +38,9 @@ You can see a list of users with accounts in your Grafana organization. If neces
 
 Update user permissions when you want to enhance or restrict a user's access to organization resources. For more information about organization permissions, refer to [Organization roles]({{< relref "../../roles-and-permissions/#organization-roles" >}}).
 
+> **Note:** As of Grafana 9.5, we have introduced a feature toggle `onlyExternalOrgRoleSync` disabling the ability for synced users to change their organizational roles during a login session.
```

Comment:
Will the org administrator be able to change that feature toggle or is that only something a sys admin can do? I'm asking because the topic you want to add this to is written for org admins, so they might think they have permission to change the toggle.

Reply:
Only the sys admin would be able to change these feature toggles, it's only available in the configuration file or via a support ticket to support on cloud.